alacritty_terminal/src/tty/windows/conpty.rs

```diff
@@ -20,9 +23,63 @@ use crate::event::{OnResize, WindowSize};
 use crate::tty::windows::child::ChildExitWatcher;
 use crate::tty::windows::{cmdline, win32_string, Pty};
 
+/// Load the pseudoconsole API from conpty.dll if possible, otherwise use the
+/// standard Windows API. The conpty.dll from the Windows Terminal project
+/// supports loading OpenConsole.exe, which offers many improvements and
+/// bugfixes compared to the standard conpty that ships with Windows.
+///
+/// In particular, the new conpty.dll and OpenConsole.exe enable mouse
+/// reporting, which not yet supported by the Windows API on Windows 10 as of
+/// June 2023.
+///
+/// The conpty.dll and OpenConsole.exe files will be searched in PATH and in
+/// the directory where Alacritty executable is located.
+struct ConptyApi {
+    create: unsafe extern "system" fn(COORD, HANDLE, HANDLE, u32, *mut HPCON) -> HRESULT,
+    resize: unsafe extern "system" fn(HPCON, COORD) -> HRESULT,
+    close: unsafe extern "system" fn(HPCON),
+}
+
+impl ConptyApi {
+    /// Try loading ConptyApi from conpty.dll library.
+    fn load_conpty() -> Option<Self> {
+        // Unsafe because of Windows API calls.
```

Comment:
This comment provides no value. If you want to make a comment on an `unsafe` block you should comment why it's **safe**. I don't need a comment to tell me an unsafe block contains unsafe code.

Reply:
OK, removing.